testutil/sims/state_helpers.go

```diff
@@ -34,30 +35,37 @@ const (
 )
 
 // AppStateFn returns the initial application state using a genesis or the simulation parameters.
-// It calls AppStateFnWithExtendedCb with nil rawStateCb.
-func AppStateFn(cdc codec.JSONCodec, simManager *module.SimulationManager, genesisState map[string]json.RawMessage) simtypes.AppStateFn {
-	return AppStateFnWithExtendedCb(cdc, simManager, genesisState, nil)
+// It calls appStateFnWithExtendedCb with nil rawStateCb.
+func AppStateFn(
```

Comment:
Ideally we would like to not change this interface so we don't break users

Reply:
This is also an "internal" function of the legacy sims and not used by any module directly

Reply:
This is a function that I have used as a developer in the past.  We should really consider any public functions as external and assume they are used by developers.  

The goal of SDK v53 is that you can update your application to the new version and have 0 API breaks